rancher/dashboard
# Add Git Repo intro page - #4952

## Summary

Issue #4830  - Update no git repo landing page.

## Steps to test
1. Navigate to Continuous delivery > Git repos.
2. Ensure there are no repos. 
2. It should display the following view of the content.
3. Note: The header and side menu is out of scope for this PR.

![image](https://user-images.githubusercontent.com/1955897/152501379-c0ea6584-fcaf-4055-be2f-8c8414d736bf.png)
